integration/spark/shared/src/testScala212/java/io/openlineage/spark/agent/lifecycle/LogicalPlanSerializer212Test.java

```diff
@@ -72,7 +76,10 @@ public boolean canEqual(Object that) {
     final ObjectMapper mapper = new ObjectMapper();
     try {
       Logger.getLogger(logicalPlanSerializer.getClass()).setLevel(Level.ERROR);
-      mapper.readTree(logicalPlanSerializer.serialize(notSerializablePlan));
+      String serialize = logicalPlanSerializer.serialize(notSerializablePlan);
+      JsonNode jsonNode = mapper.readTree(serialize);
+      assertThat(jsonNode).isNotInstanceOf(MISSING_NODE_CLASS);
+      assertThat(serialize).isEqualTo("\"<failed-to-serialize-logical-plan>\"");
```

Comment:
I return this String because it is easy to find in the code and tells you that something bad actually happen instead of staying silent.